safekeeper/Cargo.toml

```diff
@@ -47,6 +47,7 @@ pq_proto.workspace = true
 remote_storage.workspace = true
 safekeeper_api.workspace = true
 storage_broker.workspace = true
+tokio-stream.workspace = true
 utils.workspace = true
 
-workspace_hack.workspace = true
+workspace_hack.workspace = true
```

Comment:
Please add line break in the end of the file

Reply:
Done https://github.com/neondatabase/neon/pull/5731/commits/74be8c80dd94c9d6d1585ef72b234c0be129c522